libr/core/cmd_debug.c

```diff
@@ -1994,8 +1932,10 @@ static void r_core_cmd_bp(RCore *core, const char *input) {
 		bpi = r_bp_get_at (core->dbg->bp, addr);
 		if (bpi) {
 			//bp->enabled = !bp->enabled;
+			// XXX(jjd): this ^^ is what I would think toggling means...
```

Comment:
I think we discussed this the other day. Would you be against putting it back to the enabled = !enabled method instead of deleting breakpoints and recreating them when toggling ?  Otherwise it would be impossible to toggle something unless you use the address